src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
-public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+import java.util.Random;
+
+class ColorSupplier {
+
+    public Colors getRandomColor() {
+        int index = new Random().nextInt(Colors.values().length);
```

Comment:
Random should be class variable, same as in Lottery class. as we don't need to create new object each time method is called